src/Tooltip/README.md

```diff
@@ -49,3 +53,26 @@ notes: |
   ))}
 </div>
 ```
+
+### Theme variables (SCSS)
```

Comment:
Can be removed per https://github.com/edx/paragon/pull/931 (see examples in the PR).